specification/confidentialledger/resource-manager/Microsoft.ConfidentialLedger/preview/2024-03-11-preview/managedccf.json

```diff
@@ -0,0 +1,753 @@
+          {
+            "$ref": "#/parameters/ManagedCCFNameParameter"
+          },
+          {
+            "name": "ManagedCCF",
+            "in": "body",
+            "description": "Managed CCF Restore Request Body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/ManagedCCFRestore"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The request was successful; response contains a message stating the Managed CCF Resource is successfully being restored.",
+            "schema": {
+              "$ref": "#/definitions/ManagedCCFRestoreResponse"
+            }
+          },
+          "202": {
+            "description": "The Managed CCF Resource will be restored asynchronously. Check the provisioningState field to monitor the progress of your operation."
+          },
+          "default": {
+            "description": "Error response describing the reason for operation failure.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v3/types.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "ManagedCCFRestore": {
+            "$ref": "./examples/ManagedCCF_Restore.json"
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "ProvisioningState": {
+      "type": "string",
+      "description": "Object representing ProvisioningState for Managed CCF.",
+      "enum": [
+        "Unknown",
+        "Succeeded",
+        "Failed",
+        "Canceled",
+        "Creating",
+        "Deleting",
+        "Updating"
+      ],
+      "x-ms-enum": {
+        "name": "ProvisioningState",
+        "modelAsString": true
+      }
+    },
+    "RunningState": {
+      "type": "string",
+      "description": "Object representing RunningState for Managed CCF.",
+      "enum": [
+        "Active",
+        "Paused",
+        "Unknown",
+        "Pausing",
+        "Resuming"
+      ],
+      "x-ms-enum": {
+        "name": "RunningState",
+        "modelAsString": true
+      }
+    },
+    "LanguageRuntime": {
+      "type": "string",
+      "description": "Object representing LanguageRuntime for Manged CCF.",
+      "enum": [
+        "CPP",
+        "JS"
+      ],
+      "x-ms-enum": {
+        "name": "LanguageRuntime",
+        "modelAsString": true
+      }
+    },
+    "DeploymentType": {
+      "type": "object",
+      "description": "Object representing DeploymentType for Managed CCF.",
+      "title": "DeploymentType",
+      "properties": {
+        "languageRuntime": {
+          "readOnly": false,
+          "description": "Unique name for the Managed CCF.",
+          "$ref": "#/definitions/LanguageRuntime"
+        },
+        "appSourceUri": {
+          "type": "string",
+          "readOnly": false,
+          "description": "Source Uri containing ManagedCCF code"
+        }
+      }
+    },
+    "EnclaveType": {
```

Comment:
I would rename this to `EnclavePlatform` to align the naming convention with the one used by CCF.